Analysis/Core/include/AnalysisCore/CorrelationContainer.h

```diff
@@ -33,7 +34,7 @@ class CorrelationContainer : public TNamed
 {
  public:
   CorrelationContainer();
-  CorrelationContainer(const char* name, const char* objTitle, const char* reqHist = "", const char* binning = nullptr);
+  CorrelationContainer(const char* name, const char* objTitle, std::vector<o2::framework::AxisSpec> axisList);
```

Comment:
here you may consider passing only a reference to the vector to avoid copying the whole thing
`const std::vector<o2::framework::AxisSpec>&`